source/_reference-a-f.html.md.erb

```diff
@@ -2398,8 +2398,7 @@ A cell may not contain both a hyperlink and a cell link, so **hyperlink** and **
 #### Cell Value Representation
 
 [Cell objects](#cell-object) retrieved through the Smartsheet APIs have two main attributes representing cell values: **Cell.value**, and
-**Cell.displayValue**.  A third attribute, **Cell.objectValue** is currently used only for adding and updating predecessors.
-An empty cell returns no value.
+**Cell.displayValue**. A third attribute, Cell.objectValue is currently used only for adding and updating predecessors, or for multi-contact details, such as email addresses. An empty cell returns no value.
```

Comment:
Does the Cell.objectValue need the double asterisks like the one that was removed?

Reply:
good catch.